### DISCUSSION TITLE
Change color of the theme

### DISCUSSION THREAD

**Discussion by jesusfj710** [adityatelange/hugo-PaperMod#1218] [Q&A] [answered]
I was wondering if there is some doc about how to change the color of the themes (light and dark).

I know I can modify the css, with my how values, but not sure what values should I put.

Lets say I want to use the Solarized palette here. I can map de colors in if they are similars, but its does not look good.

Is there some guideline of how you can modified these or how was done for this theme? Im trying to find some information to base my modifications

Cheers!

Thanks :D

**Comment by c2tz** [comment]
Hi, I didn't find any documentation either but I invite you to have a look at this :
 [https://github.com/adityatelange/hugo-PaperMod/discussions/645](https://github.com/adityatelange/hugo-PaperMod/discussions/645) 

I haven't done it yet on my site so I can't confirm it but you just have to change the colors in relation with the name of the variables in the CSS.

Normally you just have to pay attention to the name of the variables and if necessary do some tests.

**Reply by jesusfj710** [reply]
Hey @1c2t1z! Thanks for that.

Yep, I was doing exactly that, but my concers was more in other direction.

I know the theme I want to apply. I now how to modify the colors on paperMod. The problem is that I dont know how to map them.

Let me put an example. Those are the possible vars of the theme as you share:
```
theme
entry
primary
secondary
tertiary
content
hljs-bg
code-bg
border
```

Now, I dont know how to map those vars with this ones (https://ethanschoonover.com/solarized/#the-values) since there is no description of what secondary or tertiary are used for. So, then I have to dig into all the CSS that uses those vars to discern which one are the functionality.

Sorry for my bad explanation, english is not my mother language

**Reply by c2tz** [reply, marked as answer]
So, if I understand correctly, the solarized theme you want to use is to change the colors of the `<pre><code>` (hljs) tags theme or the entire site. In any case I think there are only two solutions:

- Test 1 by 1 by changing the color by red (to see well, for example).

- Or ask the meaning of the variables to @adityatelange . Because for my part I have not yet tried.

But it is not you who explained badly, it is my brain which is a little long to understand the situation.

**Reply by jesusfj710** [reply]
No, you right. The other day I tried to match the Solarized palette with that one by "similarity" but obviusly, that went wrong.

One by one its a better option, but I found something odd. The theme is using similar the vars in a strange way. And the dark colors are duplicated in the `head.html` partial as well in the `theme-vars.css`? I dont see why, since if I modify something in the `.css` file, I wont get that in the frontend.

Anyway, I will like to dedicate more time to this, to found out why this is like this, and for that, will do what you said, contact @adityatelange. Before, I would like to do more testing to open a proper issue.

Thanks! Will mark this a solve for the moment!
